qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,134 @@
+Feature: 1.1. Search by Filters
+
+Background: 
```

Comment:
Background is not quite clear, for example, I did not catch what competence is for Data Engineer (SQL)